{"failedTests":[{"testName":"parallel/test-runner-run-watch","timestamp":1725627644138,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win10-COMPILED_BY-vs2022 #30024","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win10-COMPILED_BY-vs2022/30024/","buildNumber":30024,"builtOn":"test-azure_msft-win10_vs2019-x64-1","nodeVersion":"23.0.0","commitHash":"104de8ab88befd5340a50f10d619e0c7374042f8","callStack":[{"upstreamBuild":65609,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":74061,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":62052,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54739"}],"tap":{"ok":false,"name":"parallel/test-runner-run-watch","id":882,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"timeout"},"time":121869.871,"fullname":"parallel/test-runner-run-watch","closingTestPoint":false}},{"testName":"parallel/test-async-context-frame","timestamp":1725627644138,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2022-COMPILED_BY-vs2022 #30024","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2022-COMPILED_BY-vs2022/30024/","buildNumber":30024,"builtOn":"test-rackspace-win2022_vs2022-x64-3","nodeVersion":"23.0.0","commitHash":"104de8ab88befd5340a50f10d619e0c7374042f8","callStack":[{"upstreamBuild":65609,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":74061,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":62052,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54739"}],"tap":{"ok":false,"name":"parallel/test-async-context-frame","id":117,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"▶ AsyncContextFrame\n  ✔ async-hooks\\test-async-local-storage-args.js (4766.7249ms)\n  ✔ async-hooks\\test-async-local-storage-async-await.js (5036.975ms)\n  ✔ async-hooks\\test-async-local-storage-async-functions.js (4400.8919ms)\n  ✔ async-hooks\\test-async-local-storage-dgram.js (5281.7527ms)\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (4481.748ms)\n  ✖ async-hooks\\test-async-local-storage-enter-with.js (5168.6206ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-enter-with.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 5)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-errors.js (5255.6129ms)\n  ✖ async-hooks\\test-async-local-storage-gcable.js (4985.9967ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-gcable.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 7)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-http-agent.js (4734.4855ms)\n  ✔ async-hooks\\test-async-local-storage-http.js (5029.1467ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (4851.4945ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (4914.0067ms)\n  ✔ async-hooks\\test-async-local-storage-no-mix-contexts.js (5346.9541ms)\n  ✔ async-hooks\\test-async-local-storage-promises.js (5030.8164ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (4489.8034ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (4725.8686ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (4913.4532ms)\n  ✔ parallel\\test-async-local-storage-bind.js (4916.8494ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (4493.5879ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (4855.5609ms)\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (5161.1155ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (5875.9371ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (5385.0443ms)\n▶ AsyncContextFrame (6821.7681ms)\nℹ tests 23\nℹ suites 1\nℹ pass 21\nℹ fail 2\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 6839.9345\n\n✖ failing tests:\n\ntest at test\\parallel\\test-async-context-frame.mjs:48:5\n✖ async-hooks\\test-async-local-storage-enter-with.js (5168.6206ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-enter-with.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Promise.all (index 5)\n      at async Suite.run (node:internal/test_runner/test:1268:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }\n\ntest at test\\parallel\\test-async-context-frame.mjs:48:5\n✖ async-hooks\\test-async-local-storage-gcable.js (4985.9967ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-gcable.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Promise.all (index 7)\n      at async Suite.run (node:internal/test_runner/test:1268:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":7353.241,"fullname":"parallel/test-async-context-frame","closingTestPoint":false}}]}